extensions/packageManager/src/node/index.ts

```diff
@@ -14,8 +14,12 @@ import { FeedFactory } from './feeds/feedFactory';
 
 const API_ROOT = '/api';
 
+const hasSchema = (c) => {
+  return c.includesSchema || c.keywords?.indexOf('msbot-component') >= 0;
```

Comment:
I like this refactoring, though if `c.keywords` is an array, then I'd prefer `.includes` to the `.indexOf` thing.